src/Stripe.net/Entities/Charges/ChargeOutcomeRule.cs

```diff
@@ -2,7 +2,7 @@ namespace Stripe
 {
     using Newtonsoft.Json;
 
-    public class OutcomeRule : StripeEntity<OutcomeRule>, IHasId
+    public class ChargeOutcomeRule : StripeEntity<ChargeOutcomeRule>, IHasId
```

Comment:
So. I wanted to flag that one explicitly @ob-stripe.
It's quite an old property but, if I am not mistake, this should be a real API resource with an `object` field since this is expandable on `Charge.outcome.rule`. Really curious what your thoughts are here?

Reply:
Strictly speaking we don't _need_ the `object` field if the expandable field is not polymorphic. But AFAIK this is the only case of an expandable field where the expanded object is not an API resource with an `object` field.

I don't think it's worth fixing, but we should discuss this internally to decide if this is something we'd want to allow on other API resources in the future.

Reply:
The reason I'm asking is that, if we added `object` later the class would have the object value as a name. When right now it's kinda `ChargeOutcomeRule` and in a way it's annoying

Reply:
Do you think we should call this one `Rule` in the Radar namespace in that case so that in the future, if we add `object` it will likely be `object: "radar.rule"`?

Reply:
We should do the same thing as stripe-java to facilitate future codegen, so yes, `ChargeOutcomeRule` should be renamed to `Rule` in the Radar namespace: https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/radar/Rule.java. (Thanks for checking!)

Reply:
Fixed, PTAL